test/io/serializers/basic-types.default.good

```diff
@@ -35,6 +35,18 @@ SUCCESS
 SUCCESS
 ====================
 
+===== real(64) =====
+===== writing: =====
+2.71828
+--------------------
+2.71828
+====================
+--- read: ---
+2.71828
+-------------
+FAILURE
```

Comment:
Why does this one fail?

Reply:
Thanks for catching that, I'll fix it

Reply:
Some of the other ones do as well.  Looks like .json.good and .syntax.good